src/main/java/org/apache/bcel/classfile/StackMap.java

```diff
@@ -135,7 +135,7 @@ public StackMapEntry[] getStackMap() {
     public void setStackMap(final StackMapEntry[] table) {
         this.table = table != null ? table : StackMapEntry.EMPTY_ARRAY;
         int len = 2; // Length of 'number_of_entries' field prior to the array of stack maps
-        for (final StackMapEntry element : this.table) {
+        for (final StackMapEntry element : table) {
```

Comment:
This creates an NPE.

Reply:
Yes. I've just re-integrated the lost NPE fix.